assets/css/src/generic/_typography.scss

```diff
@@ -1,5 +1,4 @@
 #wpbody {
-  color: $color-text;
```

Comment:
I think we can't simply remove it from here as this would affect some other MailPoet plugin pages (e.g. https://d.pr/i/IBZOe0 or some texts in "help" page).
Maybe there is some common wrapper used for all MailPoet pages and we could move this definition there and somehow exclude problematic form editor page.